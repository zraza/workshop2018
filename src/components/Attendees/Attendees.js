import React from 'react';

const Attendees = ({ attendees }) => (
  <ul>
    {attendees.map(attendee => (
      <li key={attendee}>{attendee}</li>
    ))}
  </ul>
);

export default Attendees;
