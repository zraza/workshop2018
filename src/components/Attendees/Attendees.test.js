import React from 'react';
import renderer from 'react-test-renderer';

import Attendees from './Attendees';

describe('Attendees:', () => {
  const attendees = ['John Smith'];
  it('renders correctly', () => {
    const tree = renderer.create(<Attendees attendees={attendees} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
