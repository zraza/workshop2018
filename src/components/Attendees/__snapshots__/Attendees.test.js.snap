// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Attendees: renders correctly 1`] = `
<ul>
  <li>
    John Smith
  </li>
</ul>
`;
